CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ export(TARGETS libzippp NAMESPACE libzippp:: FILE ${PROJECT_BINARY_DIR}/libzippp
 if(LIBZIPPP_BUILD_TESTS)
   enable_testing()
   add_executable(libzippp_test "tests/tests.cpp")
-  target_link_libraries(libzippp_test PRIVATE libzippp)
```

Comment:
I don't get why this modification is related/needed for this PR ? :thinking:

Reply:
I've removed it